### PR TITLE
Publish design considerations

### LINKED ISSUE #1049: Publish "IIIF Design Principles" document
Design principles came up as foundational for the A/V WG charter and are also expected to inform other specification work. A draft was written by @azaroth42 a while ago but never published. Editors have agreed to get a new draft together for feedback. Style: 

  * Imperative principle headings
  * Capitalize heading words (consistent with style for sections in other docs, e.g. [Editorial Process](http://iiif.io/api/annex/notes/editors/))
  * Imperative style for the principles
  * Keep principles to short statements where possible

We'll use the set of github issues with the [principles label](https://github.com/IIIF/iiif.io/labels/principles) to discuss, one per principle. Will later assemble into document again, during draft phase (at least) linking each section back to the appropriate issue.

### PR DESCRIPTION
**Pull request by zimeon**
Creating a PR so we can discuss the specifics of getting this draft out. Fixes #1049

### CODE REVIEW

**Comment by zimeon** [comment]
I took an editing pass. Fixed a number of links and a few typos. I moved the changelog to a simple numbered section to mirror http://iiif.io/api/annex/notes/editors/#change-log , fixed copyright date, linked to specs in places mentioned (first time in any section).

**Comment by azaroth42** [comment]
Softened "decisions" to "intents" per @jpstroop's comments about them sounding defensive.

File: source/api/annex/notes/design_patterns.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+  * [Editorial Process][process]: The community process of discussion, and the editorial process of writing, the specifications.
+  * [Semantic Versioning][iiif-semver]: The interpretation of the semantic versioning pattern for managing change of APIs in a responsible and responsive manner.
+
+
+## 2. Design Principles
+
+### 2.1. Scope Design Through Shared Use Cases
+
+> [Discuss](https://github.com/IIIF/iiif.io/issues/1050)
```
Comment (jpstroop) [inline source/api/annex/notes/design_patterns.md]: I may have missed it--was the plan to keep these open and publish them?

Comment (azaroth42) [inline source/api/annex/notes/design_patterns.md]: Yes, as a nod to "this is an ongoing discussion not the editors enforcing a set of laws".

Comment (jpstroop) [inline source/api/annex/notes/design_patterns.md]: :+1:

**Comment by jpstroop** [comment]
:+1: Since it's evolving ;-)

**Comment by azaroth42** [comment]
:+1: Good to go for further discussion

**Comment by mikeapp** [comment]
👍

**Pull request merged**